README.md

```diff
@@ -700,7 +698,7 @@ Each project will have its own Page. The project page will have important detail
 <br>
 
 
-## Deployment 📦
+### Deployment 📦
```

Comment:
I think heading 2 is okay here. No need to use heading 3.

Reply:
I do think it's needed due to the header above that header (see line 92). 🙂

Reply:
https://github.com/rammcodes/Dopefolio/blob/191b3fc4e29760ade6782380641e12549ee08e38/README.md#L94
https://github.com/rammcodes/Dopefolio/blob/191b3fc4e29760ade6782380641e12549ee08e38/README.md#L706
https://github.com/rammcodes/Dopefolio/blob/191b3fc4e29760ade6782380641e12549ee08e38/README.md#L737

`## Deployment 📦` should be the same level as `## Template Instructions:` and `## Support 😇 `